src/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx

```diff
@@ -153,13 +153,17 @@ export const FeaturedCollectionsRailsContainer = createFragmentContainer(
   }
 )
 
-const Container = styled(Box)`
+const FeaturedCollectionsContainer = styled(Box)`
+  border-top: 1px solid ${color("black10")};
+`
+
+const Container = styled(Box)<{ width: string }>`
   border: 1px solid ${color("black10")};
-  width: 365px;
+  width: ${({ width }) => width};
 `
 
-const ImageContainer = styled(Flex)`
-  height: 280px;
+const ImageContainer = styled(Flex)<{ height: string }>`
+  height: ${({ height }) => height};
```

Comment:
And Flex has height: https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Flex/Flex.tsx#L20